website/docs/r/orbital_contact_profile.html.markdown

```diff
@@ -0,0 +1,164 @@
+      end_point {
+        end_point_name = "AQUA_command"
+        ip_address     = "10.0.1.0"
+        port           = "49513"
+        protocol       = "TCP"
+      }
+    }
+    direction    = "Uplink"
+    name         = "RHCP_UL"
+    polarization = "RHCP"
+  }
+  network_configuration_subnet_id = azurerm_subnet.example.id
+}
+```
+
+## Arguments Reference
+
+The following arguments are supported: 
+
+* `name` - (Required) The name of the contact profile. Changing this forces a new resource to be created.
+
+* `resource_group_name` - (Required) The name of the Resource Group where the contact profile exists.
+
+* `location` - (Required) The location where the contact profile exists.
+
+* `minimum_variable_contact_duration` - (Required) Minimum viable contact duration in ISO 8601 format. Used for listing the available contacts with a spacecraft at a given ground station.
+
+* `auto_tracking_configuration` - (Required) Auto-tracking configurations for a spacecraft. Possible values are `disabled`, `xBand` and `sBand`.
+
+* `network_configuration_subnet_id` - (Required) ARM resource identifier of the subnet delegated to the Microsoft.Orbital/orbitalGateways. Needs to be at least a class C subnet, and should not have any IP created in it.
+
+* `links` - (Required) A list of spacecraft links. A `links` block as defined below.
+
+* `event_hub_uri` - (Optional) ARM resource identifier of the Event Hub used for telemetry. Requires granting Orbital Resource Provider the rights to send telemetry into the hub.
+
+* `minimum_elevation_degrees` - (Optional) Maximum elevation of the antenna during the contact in decimal degrees.
+
+---
+
+A `links` block supports the following:
+
+* `channels` - (Required) A list of contact profile link channels. A `channel` block as defined below.
+
+* `direction` - (Required) Direction of the link. Possible values are `Uplink` and `Downlink`.
+
+* `name` - (Required) Name of the link.
+
+* `polarization` - (Required) Polarization of the link. Possible values are `RHCP`, `LHCP`, `linearVertical` and `linearHorizonal`.
+
+---
+
+A `channels` block supports the following:
+
+* `name` - (Required) Name of the channel.
+
+* `center_frequency_mhz` - (Required) Center frequency in MHz.
+
+* `bandwidth_mhz` - (Required) Bandwidth in MHz.
+
+* `end_point` - (Required) Customer End point to store/retrieve data during a contact. An `end_point` block as defined below.
+
+* `modulation_configuration` - (Optional) Copy of the modem configuration file such as Kratos QRadio. Only valid for uplink directions. If provided, the modem connects to the customer endpoint and accepts commands from the customer instead of a VITA.49 stream.
+
+* `demodulation_configuration` - (Optional) Copy of the modem configuration file such as Kratos QRadio or Kratos QuantumRx. Only valid for downlink directions. If provided, the modem connects to the customer endpoint and sends demodulated data instead of a VITA.49 stream.
```

Comment:
should this be 

```suggestion
* `modem_modulation_file_content` - (Optional) Copy of the modem configuration file such as Kratos QRadio. Only valid for uplink directions. If provided, the modem connects to the customer endpoint and accepts commands from the customer instead of a VITA.49 stream.

* `modem_demodulation_file_content` - (Optional) Copy of the modem configuration file such as Kratos QRadio or Kratos QuantumRx. Only valid for downlink directions. If provided, the modem connects to the customer endpoint and sends demodulated data instead of a VITA.49 stream.
```

Reply:
the test config seems like they are a constant?

Reply:
Same as commented above. Will double check with service team.